sklearn/metrics/_classification.py

```diff
@@ -763,10 +793,13 @@ def jaccard_score(
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
-    zero_division : "warn", {0.0, 1.0}, default="warn"
+    zero_division : "warn", 0.0, 1.0, np.nan, default="warn"
```

Comment:
Here and after:

```suggestion
zero_division : "warn", {0.0, 1.0, np.nan}, default="warn"
```

We can consider that this is a set of the possible value to pass.

Reply:
I will assume you meant `zero_division : {"warn", 0.0, 1.0, np.nan}, default="warn"`